src/BugBundle/Entity/User.php

```diff
@@ -51,16 +64,29 @@ class User implements UserInterface, \Serializable
      **/
     private $roles;
 
-
+    /**
+     * @Assert\File(maxSize="6000000")
+     */
+    private $ava;
 
     public function __construct()
     {
         $this->isActive = true;
-        $this->roles=new ArrayCollection();
+        $this->roles = new ArrayCollection();
         // may not be needed, see section on salt below
         // $this->salt = md5(uniqid(null, true));
     }
 
+    public function __toString()
+    {
+        return $this->getAnyName() ? $this->getAnyName() : '';
+    }
+
+    private function getAnyName()
```

Comment:
if you need firstname - get it
if you need username - get it
if you want to do this in twig -> user.firstname|default(user.username)